vtk 9.4.0 fails

Attempted in https://github.com/ansys/pymapdl/pull/3602

The traceback is:

```pytb
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/plotting/visualizer.py:231: AttributeError
____________ Test_plastic_solve.test_plot_nodal_plastic_eqv_strain _____________

mapdl = <ansys.mapdl.core.mapdl_grpc.MapdlGrpc object at 0x7f7c82814910>
resume = None

    @staticmethod
    @requires("ansys-tools-visualization_interface")
    def test_plot_nodal_plastic_eqv_strain(mapdl, resume):
>       assert (
            mapdl.post_processing.plot_nodal_plastic_eqv_strain(smooth_shading=True)
            is None
        )

tests/test_post.py:843: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/post.py:3195: in plot_nodal_plastic_eqv_strain
    return self._plot_point_scalars(
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/misc.py:435: in wrapper
    return function(self, *args, **kwargs)
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/post.py:635: in _plot_point_scalars
    return pl.show(**kwargs)
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/plotting/visualizer.py:842: in show
    self._backend.show()
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/tools/visualization_interface/backends/pyvista/pyvista.py:436: in show
    self.enable_widgets()
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/tools/visualization_interface/backends/pyvista/pyvista.py:186: in enable_widgets
    self._widgets.append(MeshSliderWidget(self))
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/tools/visualization_interface/backends/pyvista/widgets/mesh_slider.py:55: in __init__
    self._meshes = self.plotter_helper._pl.scene.meshes
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pyvista/plotting/plotter.py:7162: in meshes
    return [actor.mapper.dataset for actor in self.actors.values() if hasattr(actor, 'mapper')]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

.0 = <dict_valueiterator object at 0x7f7c780f4d10>

>   return [actor.mapper.dataset for actor in self.actors.values() if hasattr(actor, 'mapper')]
E   AttributeError: 'NoneType' object has no attribute 'dataset'
```

--- a/doc/changelog.d/209.fixed.md
+++ b/doc/changelog.d/209.fixed.md
@@ -1,0 +1,1 @@
+fix: not checking if actors have datasets.

--- a/src/ansys/tools/visualization_interface/backends/pyvista/widgets/mesh_slider.py
+++ b/src/ansys/tools/visualization_interface/backends/pyvista/widgets/mesh_slider.py
@@ -52,9 +52,20 @@ class MeshSliderWidget(PlotterWidget):
         self._button: vtkButtonWidget = self.plotter_helper._pl.scene.add_checkbox_button_widget(
             self.callback, position=(45, 60), size=30, border_size=3
         )
-        self._meshes = self.plotter_helper._pl.scene.meshes
         self._mb = None
         self._mesh_actor_list = []
+
+    @property
+    def _meshes(self):
+        """Return all the meshes which have dataset from the underlying plotter."""
+        # This method is patching #208
+        # until pyvista fix this upstream.
+        actors = self.plotter_helper._pl.scene.actors.values()
+        meshes = []
+        for actor in actors:
+            if hasattr(actor, 'mapper') and hasattr(actor.mapper, "dataset"):
+                meshes.append(actor.mapper.dataset)
+        return meshes
 
     def callback(self, state: bool) -> None:
         """Remove or add the mesh slider widget actor upon click.

